holoviews/core/spaces.py

```diff
@@ -146,6 +202,11 @@ def __mul__(self, other):
                     items.append((new_key, Overlay([other[other_key]])))
             return self.clone(items, kdims=dimensions, label=self._label, group=self._group)
         elif isinstance(other, self.data_type):
+            if isinstance(self, DynamicMap):
+                from .operation import DynamicOperation
+                def dynamic_mul(element):
+                    return element * other
+                return DynamicOperation(self, dynamic_mul)
```

Comment:
Prefer to use a proper function because at least it is named. Don't really care though.